sklearn/utils/__init__.py

```diff
@@ -1185,3 +1185,98 @@ def is_abstract(c):
     # itemgetter is used to ensure the sort does not extend to the 2nd item of
     # the tuple
     return sorted(set(estimators), key=itemgetter(0))
+
+
+def _empty_metadata_request():
+    return Bunch(fit={},
+                 predict={},
+                 transform={},
+                 score={},
+                 split={},
+                 fit_transform={})
```

Comment:
I suspect it's Pipeline's job to combine fit and transform metadata, rather than having a separate key for fit_transform.